en/docs/setup/configuring-a-seperate-session-store.md

```diff
@@ -0,0 +1,216 @@
+# Configuring a Separate JDBC Session Store
+
+WSO2 identity server uses the identity database as the session store.
+This document will guide to you to configure a separate JDBC session store.
+By default, the WSO2 Identity Server embedded H2
+database contains session store tables.
+
+##Configuring the internal database as session store.
+
+Add the following configuration to `<IS-HOME>/repository/conf/deployment.toml`.
+
+``` toml
+[session]
+data_source="jdbc/WSO2SessionDB"
+[datasource.WSO2SessionDB]
+id = "WSO2SessionDB"
+url = "jdbc:h2:./repository/database/WSO2IDENTITY_DB;DB_CLOSE_ON_EXIT=FALSE;LOCK_TIMEOUT=60000"
+username = "wso2carbon"
+password = "wso2carbon"
+driver = "org.h2.Driver"
+[datasource.WSO2SessionDB.pool_options]
+defaultAutoCommit = true
+maxActive = 50
+maxWait = 60000
+jmx_enable = false
+validationInterval = 30000
+```
+
+##Configuring an external database as JDBC user store.
+
+1. Create a database on [any supported RDBMS database](../../setup/working-with-databases).
+
+2. Following are the example configurations for each database type.
+
+??? example "PostgreSQL"
+
+    1. deployment.toml Configurations.
+        ```
+        [session]
+        data_source="jdbc/WSO2SessionDB"
+        [datasource.WSO2SessionDB]
+        url = "jdbc:postgresql://localhost:5432/sessiondb"
+        username = "root"
+        password = "root"
+        driver = "org.postgresql.Driver"
+        ```
+
+    2. Executing database scripts. 
+
+        Navigate to `<IS-HOME>/dbscripts/identity/postgresql.sql`. Execute the scripts of following tables
```

Comment:
Navigate to `<IS-HOME>/dbscripts/identity/postgresql.sql`. Execute the scripts in the following tables.

Change wherever applicable.